pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -31,7 +31,9 @@ func stopTray() error {
 	 * this tries to stop the old tray process, can be removed after a  few releases
 	 */
 	_, _, _ = powershell.Execute(`Stop-Process -Name tray-windows`)
-
+	if err := checkIfTrayRunning(); err != nil {
```

Comment:
Might be nice to extend the error reporting to be able to differentiate between "tray is not running" and "some other failure occurred" (no need for changes in this PR)